docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Similar to what I just added to #1695, I think the following could work with some reflection (and auto-quoting, aggressive defaulting, etc.):
```
kubectl create --app=guestbook service rc podtemplate image:dockerfile/redis
```
I'd be ok if the initial implementation weren't fully generic, but I don't want to start with CLI syntax that isn't extensible to the full realization. That means, for instance, that command-line flags corresponding to non-metadata API fields are out.

I'd love for someone to pick up #1695 and get an initial version into kubectl.